opts/opts.go

```diff
@@ -515,3 +516,21 @@ func (m *MemSwapBytes) UnmarshalJSON(s []byte) error {
 	b := MemBytes(*m)
 	return b.UnmarshalJSON(s)
 }
+
+type OomScoreAdj int64
+
+func (o *OomScoreAdj) Type() string { return "int64" }
+
+func (o *OomScoreAdj) Value() int64 { return int64(*o) }
+
+func (o *OomScoreAdj) String() string {
+	return strconv.FormatInt(int64(*o), 10)
+}
+
+func (o *OomScoreAdj) Set(value string) error {
+
+	var conv int64
+	conv, _ = strconv.ParseInt(value, 10, 64)
+	*o = OomScoreAdj(conv)
+	return nil
+}
```

Comment:
There already exists a generic `Uint64Opt` type in the `cli/command/service` package. No need to define a new one.

Reply:
That's an Unsigned 64 bit integer

Reply:
Right, my mistake. You don't need any custom type at all! https://pkg.go.dev/github.com/spf13/pflag#FlagSet.IntVar